src/extension.ts

```diff
@@ -49,6 +48,29 @@ function initializeExtension(operationId: string, context: vscode.ExtensionConte
     context.subscriptions.push(instrumentAndRegisterCommand("JavaDebug.SpecifyProgramArgs", async () => {
         return specifyProgramArguments(context);
     }));
+    context.subscriptions.push(instrumentAndRegisterCommand("java.debug.hotCodeReplace", async (args: any) => {
+        const autobuildConfig: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration("java.autobuild");
+        if (!autobuildConfig.enabled) {
+            const ans = await vscode.window.showWarningMessage(
+                "The hot code replace feature requires you to enable the autobuild flag, do you want to enable it?",
+                "Yes", "No");
+            if (ans === "Yes") {
+                await autobuildConfig.update("enabled", true);
```

Comment:
What happen after enable it?

Reply:
current hcr doesn't take effect, you need click again.